ui/ember-cli-build.js

```diff
@@ -12,6 +12,10 @@ module.exports = function(defaults) {
     svg: {
       paths: ['public/images/icons'],
     },
+    codemirror: {
+      themes: ['solarized'],
```

Comment:
Are you using this? I think if you want to, you'd still have to add the vendor CSS to your `ember-cli-build.js` file (or is auto-import that smart?!)

Reply:
Ah whoops. I am not. I'll delete it.